test/integration/tests/abrmd_policysigned.sh

```diff
@@ -102,4 +102,34 @@ tpm2 flushcontext session.ctx
 diff secret.dat unsealed.dat
 rm -f unsealed.dat
 
-exit 0
+#
+# Test with cpHashA with ECDSA signature
+#
+openssl ecparam -name prime256v1 -genkey -noout -out signing_key.priv
+openssl ec -in signing_key.priv -outform PEM -pubout -out signing_key.pub
+tpm2 loadexternal -C o -G ecc -u signing_key.pub -c signing_key_pub.ctx
+
+## Create cpHash and policy digest
+tpm2 dictionarylockout -c --cphash cphash.bin
+tpm2 startauthsession -S session.ctx
+tpm2 policysigned -S session.ctx -c signing_key_pub.ctx -L policy.signed --cphash cphash.bin
+tpm2 flushcontext session.ctx
+
+## Set lockout hierarchy authValue and policyAuth
+tpm2 changeauth -c l "password"
+tpm2 setprimarypolicy -C l -P "password" -L policy.signed -g sha256
+
+tpm2 startauthsession -S session.ctx --policy-session
+### Generate signature with cpHashA
+tpm2 policysigned -S session.ctx -c signing_key_pub.ctx --raw-data to_sign.bin --cphash cphash.bin
+openssl dgst -sha256 -sign signing_key.priv -out signature.dat to_sign.bin
+### Satisfy policy
+tpm2 policysigned -S session.ctx -g sha256 -s signature.dat -f ecdsa -c signing_key_pub.ctx --cphash cphash.bin
```

Comment:
use a continuation marker and break the line at 80 character width.

Reply:
Agreed.